lisa/tests/staging/sched_android.py

```diff
@@ -190,7 +190,7 @@ def test_stune_frequency(self, freq_margin_pct=10) -> ResultBundle:
         # into a real OPP
         boost = self.boost
         target_freq = min(max_freq, max_freq * boost / 80)
-        target_freq = list(filter(lambda f: f >= target_freq, freqs))[0]
+        target_freq = sorted(list(filter(lambda f: f >= target_freq, freqs)))[0]
```

Comment:
Methinks we should make sure `self.plat_info['freqs'][cpu]` is always sorted - either when we fetch them or in devlib directly.

Looking at this, I think `UserspaceSanity.test_performance_sanity()` suffers from the same problem: it does kinda expect we ran the workload on increasing frequencies.

Reply:
+1, let's sort self.plat_info['freqs'] or target.cpufreq.list_frequencies(cpu) in devlib:
https://github.com/ARM-software/lisa/blob/master/lisa/platforms/platinfo.py#L142